dask/dataframe/tests/test_dataframe.py

```diff
@@ -3995,10 +4003,6 @@ def test_to_datetime():
         dd.to_datetime(ds.index, infer_datetime_format=True),
         check_divisions=False,
     )
-    assert_eq(
-        pd.to_datetime(s, utc=True),
-        dd.to_datetime(ds, utc=True),
-    )
```

Comment:
This change seems unrelated.

Reply:
Yeah, I didn't change it. maybe a outdated fork